container/kvm/container.go

```diff
@@ -35,30 +35,32 @@ func (c *kvmContainer) Name() string {
 	return c.name
 }
 
-func (c *kvmContainer) Start(params StartParams) error {
+// EnsureCachedImage ensures that a container image suitable for satisfying
+// the input start parameters has been cached on disk.
+func (c *kvmContainer) EnsureCachedImage(params StartParams) error {
 	var srcFunc func() simplestreams.DataSource
 	if params.ImageDownloadURL != "" {
 		srcFunc = func() simplestreams.DataSource {
 			return imagedownloads.NewDataSource(params.ImageDownloadURL)
 		}
 	}
-	var ftype = BIOSFType
+	var fType = BIOSFType
```

Comment:
Sorry about the noise from these - just drive-by appeasements of the spell checker, and the linter complaining about import name collisions.

Reply:
Why don't you split the
- fType renames
- the _ = assignment changes
- shortening of container to c
- the logDir renames

Into one commit each?
That would make the actual change that is meant to be done (synchronize image activities) much more reviewable.

Reply:
Thanks; that is generally my approach.